dashboard/requirements.in

```diff
@@ -1,6 +1,28 @@
-jsonschema
-requests
-matplotlib
+pycodestyle
```

Comment:
You have pydocstyle twice

Reply:
Will remove

Reply:
No pydocstyle and pycodestyle are different